yt/go/yt/internal/rpcclient/encoder.go

```diff
@@ -1155,6 +1156,35 @@ func (e *Encoder) AddMember(
 	return
 }
 
+func (e *Encoder) SetUserPassword(
+	ctx context.Context,
+	user string,
+	newPassword string,
+	currentPassword string,
+	options *yt.SetUserPasswordOptions,
+) (err error) {
+	return xerrors.Errorf("Unimplemented method: SetUserPassword")
```

Comment:
Do you know, if these methods are implemented in RPC proxy API on the server side?

Reply:
fixed

Reply:
I don't, I've assumed that `Unimplemented` here means `not implemented in go client`.

Reply:
But i suppose they [don't](https://github.com/l0kix2/ytsaurus/blob/3b44ede92c2644c9f0465908b512accb87527c5f/yt/yt/client/api/rpc_proxy/client_impl.cpp#L2041-L2074).